### PR TITLE
Package data-encoding.0.1.1

### PR DESCRIPTION
**Pull request by raphael-proust**
### `data-encoding.0.1.1`
Library of JSON and binary encoding combinators

---
* Homepage: https://gitlab.com/nomadic-labs/data-encoding
* Source repo: git+https://gitlab.com/nomadic-labs/data-encoding.git
* Bug tracker: https://gitlab.com/nomadic-labs/data-encoding/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/data-encoding/data-encoding.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" ]
+homepage: "https://gitlab.com/nomadic-labs/data-encoding"
+bug-reports: "https://gitlab.com/nomadic-labs/data-encoding/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/data-encoding.git"
+license: "MIT"
+depends: [
+  "ocaml" { >= "4.07" }
+  "dune" { >= "1.7" }
```
Comment (kit-ty-kate) [inline packages/data-encoding/data-encoding.0.1.1/opam]:
```suggestion
  "dune" { >= "1.11" }
```
According to the `dune-project` file this packages requires at least dune 1.11

Comment (raphael-proust) [reply packages/data-encoding/data-encoding.0.1.1/opam]: I went the other way around and downgraded `dune-lang` in `dune-project`.

Comment (kit-ty-kate) [reply packages/data-encoding/data-encoding.0.1.1/opam]: Sure, that also works!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**